lib/travis/api/v3/billing_client.rb

```diff
@@ -89,7 +89,7 @@ def handle_errors_and_respond(response)
     end
 
     def connection
-      @connection ||= Faraday.new(url: billing_url, ssl: { ca_path: '/usr/lib/ssl/certs' }) do |conn|
+      @connection ||= Faraday.new(url: billing_url, ssl: Travis.config.ssl.to_h.merge({})) do |conn|
```

Comment:
It changes current implementation.
It's not good solution.

Reply:
Did you take into consideration cloud solution? What settings are for travis-ci.org and travis-ci.com? Will it work?

Reply:
No, it will not work on cloud.
Take a note that you have change default:
 ca_file: '/etc/ssl/certs/cacert.pem'
which was previously used here as: '/usr/lib/ssl/certs'